Tests/SwiftParserTest/Statements.swift

```diff
@@ -45,8 +45,7 @@ final class StatementTests: XCTestCase {
       }
       """,
       diagnostics: [
-        DiagnosticSpec(message: "expected expression in pattern"),
-        DiagnosticSpec(message: "expected '=' and expression in pattern matching"),
+        DiagnosticSpec(message: "expected expression, '=' and expression in pattern matching"),
```

Comment:
Might be worth adding some more fix-it checks? Is this one`<#expression#> = <#expression#>`?

Reply:
I think I didn’t bother adding these checks for this test case because it’s so broken that we’ll have a hard time producing a good diagnostic (let alone Fix-Its). I don’t even know what I would expect here. I can add more checks if you prefer though.

Reply:
I didn't really mean this case specifically. Just that you added merging for more, but no checks that the generated fix-it in these cases made any sense. I suspect it should since it hasn't really change, would just be nice to make sure.

I was mostly wondering whether we do in fact use editor placeholders for "expected expression".